packages/select/src/common/listItemsProps.ts

```diff
@@ -22,6 +33,21 @@ export interface IListItemsProps<T> extends IProps {
     /** Array of items in the list. */
     items: T[];
 
+    /**
+     * Specifies how to test if two items values are equivalent.
+     *
+     * If omitted or `undefined`, then simple strict equality is used by default.
+     *
+     * Provide an equality comparator function (that returns true if the two params are equivalent), or
+     * simply provide the name of a property on the item that exposes whose value
+     * can be compared with strict equality to determine equivalence (e.g., a string/number ID property)
+     *
+     * If a equality comparator function is provided, its params values will never be `null` or `undefined`.
+     * Comparisons involving `null` and `undefined` values are handled automatically without calling
+     * the equality comparator function.
```

Comment:
nice caveat! good works on the docs here, very thorough. i have edited for clarity, what do you think?

    /**
     * Specifies how to test if two items are equal. By default, simple strict
     * equality (`===`) is used to compare two items.
     *
     * If your items have a unique identifier field, simply provide the name of
     * a property on the item that can be compared with strict equality to
     * determine equivalence: `itemsEqual="id"` will check `a.id === b.id`.
     *
     * If more complex comparison logic is required, provide an equality
     * comparator function that returns `true` if the two items are equal. The
     * arguments to this function will never be `null` or `undefined`, as those
     * values are handled before calling the function.
     */

Reply:
I like your version better. Documentation is hard :)

Reply:
it sure is! and i have written _a lot_ of it. i'm always happy to edit docs in PRs like this 👍

Reply:
Done